src/client/sandbox/code-instrumentation/properties/index.js

```diff
@@ -28,6 +28,18 @@ import { emptyActionAttrFallbacksToTheLocation } from '../../../utils/feature-de
 
 const ORIGINAL_WINDOW_ON_ERROR_HANDLER_KEY = 'hammerhead|original-window-on-error-handler-key';
 
+function checkoutAttrs (element) {
```

Comment:
May be `checkElementTextProperties`?
Because we check list of properties https://github.com/DevExpress/testcafe-hammerhead/pull/1166/files/06cec7bc260014bbaef58a672d04162c36421fa4#diff-8a9510bf5230ab2620f64db8801899c9R35